kinematic_params:
  a1: 0.5    #[m]
  a2: 0.5    #[m]
  

  # General Link/Joint Parameters
  link_radius:  0.03      #[m]
  
  joint_limit:  1.0e+16   #[deg]
  joint_effort: 30.0
  joint_vel:    2.0

  base_width:   0.2       #[m]
  base_height:  0.50       #[m]
  #parameters of arm model
  arm_length:   0.35      #[m] total lenght in x direction
  arm_width:    0.15      #[m] total widht in y direction
  arm_height:   0.03      #[m] thickness in z direction | depending on .stl 3D model!
  arm_dist_rot:  0.25      #[m] length in x dsirection between two rotation origins  
  #scales of arms
  scalex_1:     1.8
  scalex_2:     1.5

dynamic_params:
  m1: 1.0         #[kg]
  m2: 1.0         #[kg]
  m3: 1.0
  base_mass: 1.0   #[kg]

simulation:
  fs: 1000          #[Hz]
  q1_init:  0.01    #[rad]
  q2_init:  -0.07   #[rad]

imu:
  mass: 0.001
  ixx: 0.0001
  iyy: 0.000001
  izz: 0.0001
  size: 0.001
  freq: 100.0
  noise: 0.001